.drone.jsonnet

```diff
@@ -17,8 +17,5 @@ local Pipeline(os, arch, version) = {
 };
 
 [
-    Pipeline("linux", "arm",   "1.3"),
-    Pipeline("linux", "arm64", "1.0"),
-    Pipeline("linux", "arm64", "1.2"),
-    Pipeline("linux", "arm64", "1.3")
+    Pipeline("linux", "arm64", "1.3"),
```

Comment:
Should work also on `arm` with Julia 1.3, thanks to https://github.com/JuliaLang/julia/pull/32803.  Needs to update the `.drone.yml`, too.